src/library_glemu.js

```diff
@@ -3306,7 +3376,24 @@ var LibraryGLEmulation = {
 
   glPolygonMode: function(){}, // TODO
 
-  glAlphaFunc: function(){}, // TODO
+  glAlphaFunc: function(func, ref) {
+    switch(func) {
+      case 0x200: // GL_NEVER
+      case 0x201: // GL_LESS
+      case 0x202: // GL_EQUAL
+      case 0x203: // GL_LEQUAL
+      case 0x204: // GL_GREATER
+      case 0x205: // GL_NOTEQUAL
+      case 0x206: // GL_GEQUAL
+      case 0x207: // GL_ALWAYS
+        GLEmulation.alphaTestRef = ref;
+        if (GLEmulation.alphaTestFunc != func) {
+          GLEmulation.alphaTestFunc = func;
+          GLImmediate.currentRenderer = null; // alpha test mode is part of the FFP shader state, we must re-lookup the renderer to use.
+        }
+        break;
+    }
```

Comment:
Perhaps add a default with a warning in `GL_ASSERTIONS` mode about any other value here?